### DISCUSSION TITLE
better point and reward plugin

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5559] [New Feature/Ideas Requests]
**[https://yithemes.com/themes/plugins/yith-woocommerce-points-and-rewards/](https://yithemes.com/themes/plugins/yith-woocommerce-points-and-rewards/)**
![00 YITH-WooCommerce-Points-Rewards](https://user-images.githubusercontent.com/57155882/174439642-218aca86-9dc5-4259-9a1e-6292a3e221f2.jpg)
![06 YITH-WooCommerce-Points-Rewards-Gamification-Rank-Customers-Points](https://user-images.githubusercontent.com/57155882/174439649-ce133ec8-78f7-41eb-9ef0-a950e09c2396.jpg)
![05 YITH-WooCommerce-Points-Rewards-Levels-Badges-Gamification](https://user-images.githubusercontent.com/57155882/174439651-cc99ad80-20bb-4f79-b48b-81fd64fcf12b.jpg)
![07 YITH-WooCommerce-Points-Rewards-Gamifications-Banner-Get-Points](https://user-images.githubusercontent.com/57155882/174439654-71f20875-2447-44b3-8805-93250c89a057.jpg)

**Comment by osman-kocak** [comment]
@minhcasi @luyx2412 @trunghieuvn

**Comment by Yan-Aleman** [comment]
Cool Plugin.

**Comment by HanielQC** [comment]
Dear @osman-kocak , @Yan-Aleman , 
We are planning to develop FluxStore to support https://yithemes.com/themes/plugins/yith-woocommerce-points-and-rewards/?_1706790612
Could you please share your admin account by emailing hi@inspireui.com so our technical team can develop it? Thank you

**Reply by Yan-Aleman** [reply]
Hello.  I am using "woocommerce points and rewards" from WooCommerce.  https://woocommerce.com/products/woocommerce-points-and-rewards .  And with the plugin "yith" I only got acquainted with the documentation, in this post)
__
but if I purchase this plugin, I will gladly share it.
